packages/amplify_api_rest/README.md

```diff
@@ -0,0 +1,15 @@
+# amplify_api_rest
+
+A new flutter plugin project.
```

Comment:
Don't forget to update the readme.  See other packages for examples.

Reply:
will update in our merged package (RestAPI + GraphQL)